media/css/firefox/whatsnew/whatsnew-107-en.scss

```diff
@@ -0,0 +1,116 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+$font-path: '/media/protocol/fonts';
+$image-path: '/media/protocol/img';
+
+@import '~@mozilla-protocol/core/protocol/css/components/notification-bar';
+@import '~@mozilla-protocol/core/protocol/css/components/section-heading';
+@import '~@mozilla-protocol/core/protocol/css/includes/lib';
+@import 'includes/base';
+@import 'includes/dark-mode';
+
+// * -------------------------------------------------------------------------- */
+// Main content
+.wnp-content-main {
+    padding-top: $spacing-md;
+    text-align: center;
+}
+
+.wnp-main-title {
+    @include text-title-md;
+}
+
+.wnp-sub-title {
+    @include text-title-md;
+    color: #a77ffa;
+}
+
+.wnp-main-img {
+    max-width: $content-md;
+    margin: $layout-sm auto $layout-lg;
+
+    svg {
+        width: 100%;
+        height: auto;
+    }
+}
+
+.wnp-main-tagline {
+    @include text-body-lg;
+    max-width: $content-md;
+    margin: $spacing-xl auto;
+}
+
+.wnp-main-cta {
+    margin-top: $spacing-xl;
+}
+
+.c-coupon {
+    background: #c3c4fd;
+    border-radius: $border-radius-sm;
+    color: $color-black;
+    margin: $layout-lg auto 0;
+    max-width: $content-md;
+    padding: $spacing-xl;
+
+    h3 {
+        @include text-title-2xs;
+        color: $color-black;
+        margin-bottom: $spacing-lg;
+    }
+
+    .c-disclaimer {
+        @include text-body-xs;
+        margin-bottom: 0;
+    }
+
+    .c-coupon-code {
+        @include text-body-xl;
+        border-radius: $border-radius-sm;
+        border: 1px solid $color-black;
+        display: inline-block;
+        font-weight: normal;
+        padding: $spacing-sm $spacing-md;
+    }
+
+    .c-coupon-copy {
```

Comment:
**nit (non-blocking)**: top vertical spacing feels a bit tight, touch space is good horizontally but could be a little more generous vertically

<img width="276" alt="copy-spacing" src="https://user-images.githubusercontent.com/19650432/200860918-8fd77818-70c0-40ec-a409-8acd632176a0.png">

**suggestion**: add vertical padding (or at least top to space out the Copy Code text a bit more)